admin_guide/out_of_resource_handling.adoc

```diff
@@ -13,26 +13,27 @@ toc::[]
 == Overview
 
 The node must preserve node stability when available compute resources are
-low. This is especially important when dealing with incompressible resources
-such as memory or disk. If either resource is exhausted, the node becomes
-unstable.
+depleting. This is especially important when dealing with incompressible
+resources such as memory or disk. If either resource is exhausted, the node
+becomes unstable.
 
 [[out-of-resource-eviction-policy]]
 == Eviction Policy
 
-The node can proactively monitor for and prevent against total starvation of a
-compute resource. In cases where this appears to occur, the node can
-proactively fail one or more pods in order to reclaim the starved resource.
-When the node fails a pod, it terminates all containers in the pod, and the
-`*PodPhase*` is transitioned to *Failed*.
+Using eviction policies, a node can proactively monitor for and prevent
+against total starvation of a compute resource.
+
+In cases where a node is losing resources, it can proactively fail one or more
+pods in order to reclaim the starved resource using an eviction policy. When the
+node fails a pod, it terminates all containers in the pod, and the `*PodPhase*`
+is transitioned to *Failed*.
 
 Platform administrators can configure eviction settings within the
 *_node-config.yaml_* file.
-xref:out-of-resource-schedulable-resources-and-eviction-policies[See an example
-scenario].
 
 [[out-of-resource-eviction-signals]]
 === Eviction Signals
```

Comment:
I do think this could be changed to something like "How the node signals that it's almost full", (though that's a terrible suggestion). But really, the paragraph below seems to be describing a setting that says that. Any other suggestions?